### PR TITLE
Addng new Article for 2016-11 - Hacker Herding

### PR DESCRIPTION
**Pull request by sbg-danadams**
Addng new Article for 2016-11 - Hacker Herding

### CODE REVIEW

**Comment by WheresAlice** [comment]
The category should be capitalised (Security).  The title in the frontmatter either needs quotes around it or the colon removing.  With those two fixed all the tests in my upcoming rspec validator tool pass.

File: _posts/2016-11-17-herding-hackers.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+## Tip 5 - Start small, start slick
+Its easy to get tempted into starting a large public bounty program, but I would strongly advise you to smart small and scale slow -try testing the waters with an internal/private bug bounty programme, or a fixed PoC/pilot with a set bounty pot and a small number of hand-picked hackers. Its also vital to have a dedicated budget agreed and put aside, with the ability to quickly clear funds. Since no pot of money is infinite, its important also to limit your exposure somehow and maintain control over likely expenditure. It may be a good idea, at least initially to run for a defined trial period, or structure your programme around bursts, sprints or "seasons" so that you have an on-season and an off-season for bug hunting, giving you time in between to clear down vulnerability backlogs and ready for the next onslaught.
```
Comment (tomnomnom) [inline _posts/2016-11-17-herding-hackers.md]: s/Its/It's/

**Comment by sbg-danadams** [comment]
Cheers Alice. Fixed.

Daniel Adams CISSP
Web Application Security Specialist
TEL:   +44 (0)  7725 208879
EMAIL: daniel.adams@skybettingandgaming.com<mailto:daniel.adams@skybettingandgaming.com>

Upcoming leave: [none]

[https://tools.skybet.net/confluence/s/en_GB/5989/aaad9997c145089d7f38b9dea0ac5b91728ef55a.15/_/images/logo/confluence-logo.png]

File: _posts/2016-11-17-herding-hackers.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## Tip 5 - Start small, start slick
+Its easy to get tempted into starting a large public bounty programme, but I would strongly advise you to smart small and scale slow - try testing the waters with an internal/private bug bounty programme, or a fixed PoC/pilot with a set bounty pot and a small number of hand-picked hackers. It's also vital to have a dedicated budget agreed and put aside, with the ability to quickly clear funds. Since no pot of money is infinite, it's important also to limit your exposure somehow and maintain control over likely expenditure. It may be a good idea, at least initially, to run for a defined trial period, or structure your programme around bursts, sprints or "seasons" so that you have an on-season and an off-season for bug hunting, giving you time in between to clear down vulnerability backlogs and get ready for the next onslaught.
```
Comment (swestcott) [inline _posts/2016-11-17-herding-hackers.md]: s/smart small/start small/

Comment (sbg-danadams) [inline _posts/2016-11-17-herding-hackers.md]: Thanks Si, fixed now

File: _posts/2016-11-17-herding-hackers.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+## Tip 8 - Management lives matter too
+Its easy to get swept up in the enthusiasm of operating a bug bounty programme but there may be a long list of people that you need to get support from for buy-in both at launch and on an ongoing basis. Are senior management happy at the increased risk profile of operating a bug bounty programme? What about your external comms team? Legal teams? How about your product owners? Do your network team know why they're seeing a sudden uplift in XSS probes on the web application firewall? Have you made your compliance teams aware? You're going to want to engage various teams early as possible, and feed back with the wonderful metrics you're gathering (see Tip 3) regularly.
+
+## Tip 9 - Partner Up
+Stick to what you're good at and don't try and do everything yourself. Dedicated bug bounty management platforms such as HackerOne (www.hackerone.com) can prove invaluable. They're not free but they offer solid tooling as well as guidance on programme operation and services such as mediation of any challenges or disagreements with hackers.
```
Comment (medains) [inline _posts/2016-11-17-herding-hackers.md]: Linking stuff:
Not ```HackerOne (www.hackerone.com)```
But instead ```[HackerOne](www.hackerone.com)```

Comparison:

HackerOne (www.hackerone.com) OR  [HackerOne](www.hackerone.com)

The article as a whole would probably benefit from some images - a hackerone logo for this tip for example (official assets at https://hackerone.com/press)

Comment (sbg-danadams) [inline _posts/2016-11-17-herding-hackers.md]: Thanks Colin. The HackerOne link format is actually deliberate and I prefer the former. Don't think we have any style guidelines on this so happy to leave that one as-is

Point taken on images, I'm always loathe to try adding images due to attribution and copyright but I'll see what I can do

**Comment by sbg-danadams** [comment]
Images added now @medains

**Comment by sbg-danadams** [comment]
Feel like we're starting to obsess/nitpick at this point so would like to get this released no if no objections?

**Pull request merged**